Install.md

```diff
@@ -14,11 +13,16 @@
     LaTeX Workshop's [_"Show package documentation"_ functionality][showdoc] will therefore not work.
     (You can instead use the CTAN link, also provided in the hover popup, to download the docs).
     </details>
-    
-- We don't recommend [MiKTeX](https://miktex.org/) because MiKTeX does not ship with Perl. If you choose MiKTeX, you have to install Perl by yourself, which `latexmk` requires. Without Perl, `latexmk` fails with errors.
-- `latexmk` is required for the default recipe for building LaTeX projects to work. Alternatively, you can [set up your own LaTeX recipe](Compile#latex-recipes).
-- _Optional_: Install [ChkTeX](https://www.nongnu.org/chktex) to lint LaTeX projects.
-- _Optional_: Install [latexindent.pl](https://github.com/cmhughes/latexindent.pl) for formatting support if it is not provided by your LaTeX distribution. You also have to install a few standard Perl modules. See the [official document](https://latexindentpl.readthedocs.io/en/latest/sec-appendices.html#required-perl-modules).
+
+  - [MiKTeX](https://miktex.org/) another light-weight distribution with a covinient automatic on-demand package install. Note however, that **for MiKTeX to work correctly with Latex Workshop, you need to install [Perl](https://strawberryperl.com/)**.
```

Comment:
covinient -> convinient

Reply:
Sorry, I've missed even more errors :) Fixed in the second commit.